core/common/assets/js/api/core/data.js

```diff
@@ -189,27 +195,30 @@ export default class Data extends Commands {
 	 * If the command have query convert it to args.
 	 *
 	 * @param {string} command
-	 * @param {object} argsTarget
+	 * @param {object} args
 	 *
-	 * @returns {string} command
+	 * @returns {ExtractedCommand} command
```

Comment:
Is `ExtractedCommand` a type? It seems like plain object

Reply:
Known object.

Reply:
As far as I know, only instances object should use this kind of typing. Could you provide me with a reference?

Reply:
@kobizz see the typedef above:
https://github.com/elementor/elementor/pull/11660/files#diff-d0a28344df7ce0852c5a7c797db34c66R21

Reply:
this is how jsdoc works its create 'fake' type.